debian/control

```diff
@@ -0,0 +1,20 @@
+Source: openstreetmap-cgimap
+Section: libs
+Priority: optional
+Maintainer: Matt Amos <zerebubuth@gmail.com>
+Build-Depends: debhelper (>=10), pkg-config, zlib1g-dev, libbrotli-dev, libxml2-dev, libpqxx-dev, libfcgi-dev, libboost-program-options-dev, libfmt-dev, libmemcached-dev, libcrypto++-dev, libargon2-dev, libyajl-dev, postgresql-all, postgresql-common (>= 148~), postgresql-server-dev-all, cmake
+Homepage: https://github.com/zerebubuth/openstreetmap-cgimap
+Standards-Version: 3.9.7
+
+Package: openstreetmap-cgimap-bin
```

Comment:
```suggestion
Package: openstreetmap-cgimap
```

Afaik the binary package can have the same name as the source, no need to add `-bin`

Reply:
This seems to work. You can try out a build from here: https://github.com/mmd-osm/openstreetmap-cgimap/actions/runs/10441281104

Reply:
[openstreetmap-cgimap-bin.dirs](https://github.com/mmd-osm/openstreetmap-cgimap/blob/test/449/debian/openstreetmap-cgimap-bin.dirs) and [openstreetmap-cgimap-bin.install](https://github.com/mmd-osm/openstreetmap-cgimap/blob/test/449/debian/openstreetmap-cgimap-bin.install) also need to be changed I believe

Reply:
Ok, let's give it a try...